src/DependenciesPackager/ProcessRunner.cs

```diff
@@ -96,9 +96,11 @@ public int Run()
             }
 
             var processInfo = CreateProcessInfo();
-            _process = new Process();
-            _process.StartInfo = processInfo;
-            _process.EnableRaisingEvents = true;
+            _process = new Process
+                       {
```

Comment:
Object initializer : indent at start of line or at the start of the object declaration? (i.e. - Do you want { below `_process` or `new`?